frontend/package.json

```diff
@@ -10,11 +10,14 @@
   },
   "dependencies": {
     "@types/d3": "7.4.0",
+    "@types/geojson": "^7946.0.10",
```

Comment:
what is this one used for?

Reply:
That's not needed anymore, I was trying to resolve some typing errors, I will remove it...